src/pricer/utils.ts

```diff
@@ -22,35 +23,19 @@ const tokenToApi: Record<string, ExchangeApiName[]> = {
   TKN: ['testExchange'],
 };
 
+const CACHE_EXPIRATION_TIME = 60_000;
```

Comment:
Should we make this parameter configurable?

Reply:
Im not sure about each exchange API require different timeouts, this cache was applied just to avoid consecutive queries in short intervals.